src/User.js

```diff
@@ -1,14 +1,47 @@
 class User {
-  constructor(user){
+  constructor(user) {
     this.id = user.id;
     this.name = user.name;
-    this.customer = user.customer;
     this.userBookings = [];
+    this.bookingCost = 0;
+
+  }
+  // Methods I think I'll need
+
+  retrieveRoomBookings(bookingData) {
+
+    bookingData.filter(booking => {
+      if (this.id === booking.userID) {
+        this.userBookings.push(booking);
+      }
+    });
+    // console.log(this.userBookings)
+    return this.userBookings;
+
   }
-}
 
+  calculateRoomCost(roomData) {
+    this.userBookings.forEach(bookingInput => {
+      // console.log('booking input',bookingInput);
+      roomData.filter(room => {
```

Comment:
Since the rooms will be unique by `roomNumber`, you could also use `.find` instead of `.filter` 👍